### DISCUSSION TITLE
CoPilot behavior with comment blocks in VS2022.

### DISCUSSION THREAD

**Discussion by PaulStSmith** [community/community#121899] [Copilot Conversations]
### Select Topic Area

Show & Tell

### Body

I use CoPilot a lot to comment my code, and I like to use comment blocks in a certain way, like this:

``` lang-cs
/*
 * My comment block.
 */
```

CoPilot is great to write comments, but when writing comments blocks, there is some interference from Visual Studio 2022, as shown in the very short video [here](https://youtu.be/VHk65suDsLQ).

Let's say that the block below was CoPilot's suggestion. When I press tab to accept its suggestion, the comment is entered like this:
``` lang-cs
/*
 *    * My comment block.
 *        */
```

How can we fix this?

Is there even a way?

**Comment by Akash1134** [comment]
Hi @PaulStSmith , 

You might want to look around your Visual Studio 2022 settings, specifically those for auto-completion or automatic formatting. Turning off auto-completion for comments could help the cause. Also, if you've got other extensions running that mess with code formatting or auto-completion, try disabling them to see if it resolves the issue.

Another possibility could be accepting the suggestions with TAB, have you tried accepting the Copilot's suggestion with Enter? Different keys can trigger different behaviors in the IDE.

Hope that helps!

**Comment by ayim** [comment]
Interesting - could you file a bug report via Report a Problem?
We implemented a fix but need to investigate further since it's still happening to ya.
![image](https://github.com/community/community/assets/10951869/104cd0ef-9871-4398-aaf3-e8c6a9a8fc98)

Send me the issue link once it's filed and i'll make sure to expedite.
